airbyte-integrations/connectors/source-hubspot/source_hubspot/source.py

```diff
@@ -142,8 +142,5 @@ def streams(self, config: Mapping[str, Any]) -> List[Stream]:
         return available_streams
 
     def get_custom_object_streams(self, api: API, common_params: Mapping[str, Any]):
-        schemas = api.get_custom_object_schemas()
-        streams = []
-        for entity, schema in schemas.items():
-            streams.append(CustomObject(entity=entity, schema=schema, **common_params))
-        return streams
+        for (entity, fully_qualified_name, schema) in api.get_custom_objects_metadata():
```

Comment:
Let's add some basic unit tests for the `get_custom_object_metadata()` function? I did a quick glance, I don't think we ever added any tests for `get_custom_object_schemas()` and it would be good to get a little bit of coverage on this method.

Reply:
:+1: done